packages/lab/src/dropdown-next/DropdownNext.tsx

```diff
@@ -227,19 +230,20 @@ export const DropdownNext = forwardRef(function DropdownNext(
       </button>
       <FloatingComponent
         open={open && !disabled}
-        ref={floating}
+        ref={portalRef}
         {...getDropdownNextProps()}
         {...getPosition()}
       >
         <ListNext
           id={listId}
           className={clsx(withBaseName("list"), ListProps?.className)}
           disableFocus
-          disabled={disabled || ListProps?.disabled}
+          disabled={disabled ?? ListProps?.disabled}
```

Comment:
last one, sorry i didn't catch it last time, the same wrong operator thing might be happening here